.travis.yml

```diff
@@ -40,6 +40,7 @@ matrix:
         - texlive-fonts-recommended
   - env: CI_KIND=build XARCH=x64 CONFIG_ARG=--disable-shared
   - env: CI_KIND=build XARCH=x64 MIN_BUILD=1
+  - env: CI_KIND=build XARCH=x64 CONFIG_ARG=--enable-dependency-generation MAKE_ARG=-j
```

Comment:
You can write this as:
```
  - env: >-
      CI_KIND=build
      XARCH=x64
      CONFIG_ARG=--enable-dependency-generation
      MAKE_ARG=-j
```
The `>-` means that the block should be folded into a single string, with all indentation reduced to a single space, with no embedded newlines and no newlines at the end. (See https://yaml-multiline.info/)

Reply:
Brilliant - the format that's so easy to understand that there's an entire website devoted to just that feature :joy: Thanks @stedolan! Fingers crossed Travis will correctly parse the new commit...

Reply:
Huzzah, it does!